### PR TITLE
Update Storage Files swagger to 2019-02-02

### PR DESCRIPTION
**Pull request by tg-msft**
Also adding a transform to automatically add the serialization attribute back on hierarchical blobs when regenerating.

I'd appreciate if @alzimmermsft could take over this PR and get it building/passing tests like the other swagger change.  (You can see the tweaks I made to get .NET working at https://github.com/Azure/azure-sdk-for-net/pull/7319)

### CODE REVIEW

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java [hunk 1]
```diff
@@ -29,6 +29,22 @@ public AzureFileStorageBuilder version(String version) {
         return this;
     }
 
+    /*
+     * Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.
+     */
```
Comment (rickle-msft) [inline sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java]: Why is this on the builder. This seems like a parameter that should be passed into an api

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java]: @tg-msft Do you have more insight on what this gets used for?

Comment (tg-msft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java]: It came with the new swagger:

```json
        "FileRangeWriteFromUrl": {
            "name": "x-ms-write",
            "x-ms-client-name": "fileRangeWriteFromUrl",
            "in": "header",
            "description": "Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.",
            "required": true,
            "type": "string",
            "enum": [
                "update"
            ],
            "default": "update",
            "x-ms-enum": {
                "name": "FileRangeWriteFromUrlType",
                "modelAsString": false
            }
        },
```

Comment (rickle-msft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/implementation/AzureFileStorageBuilder.java]: I am pretty sure that comes from [this api](https://docs.microsoft.com/en-us/rest/api/storageservices/put-range). In normal write operations, it's used to specify whether the operation is to update the data or clear the range. When specifying a source url (hence the "fromURL", the only viable option is update.

This should just be an internal constant that is used for the writeRangeFromURL method.

**Pull request merged**